packages/forms/src/UIForm/fields/Code/__snapshots__/Code.component.test.js.snap

```diff
@@ -21,54 +21,39 @@ exports[`Code field should render ace-editor in FieldTemplate 1`] = `
     >
       To focus out of the editor, press ESC key twice.
     </div>
-    <ReactAce
-      className="tf-widget-code form-control"
-      cursorStart={1}
-      editorProps={
-        Object {
-          "$blockScrolling": Infinity,
-        }
-      }
-      enableBasicAutocompletion={false}
-      enableLiveAutocompletion={false}
-      focus={true}
-      fontSize={12}
-      height="500px"
-      highlightActiveLine={true}
-      maxLines={null}
-      minLines={null}
-      mode=""
-      name="myCodeWidget_wrapper"
-      onBlur={[Function]}
-      onChange={[Function]}
-      onLoad={[Function]}
-      onPaste={null}
-      onScroll={null}
-      readOnly={false}
-      scrollMargin={
-        Array [
-          0,
-          0,
-          0,
-          0,
-        ]
+    <React.Suspense
+      fallback={
+        <div>
+          Loading...
+        </div>
       }
-      setOptions={
-        Object {
-          "enableBasicAutocompletion": true,
-          "enableLiveAutocompletion": true,
-          "enableSnippets": true,
+    >
+      <UNDEFINED
```

Comment:
😬

Reply:
May be just inject the global var before the tests are runned should fix that